### PR TITLE
Allow users to create free posts

### PR DESCRIPTION
**Pull request by dogpineapple**
Premium users can only make premium posts right now. This PR will allow them to create posts to be either Free or Premium posts. 

**NOTE: This PR's working branch is branched off of another working branch that was working on bug fixes: PR #31** 

### Frontend Changes
- Added `Form.Check` element into the BlogForm component.

### CODE REVIEW

File: bloggies_frontend/src/redux/rootReducer.tsx [hunk 1]
```diff
@@ -13,13 +13,16 @@ function rootReducer(state = INITIAL_STATE, action: Action) {
     case t.UPDATE_CUSTOMER_ID: 
       return {...state, user:{ ...state.user, customer_id: action.payload.customer.id}};
     case t.UPDATE_SUBSCRIPTION_ID: 
-      return {...state, user: { ...state.user, subscription_id: action.payload.subscription_id}}
+      console.log("UPDATE SUBSCRIPTION ID", action.payload);
+      return {...state, user: { ...state.user, subscription_id: action.payload.subscription_id}};
```
Comment (stefanosu) [inline bloggies_frontend/src/redux/rootReducer.tsx:16]: I see the console log is here to check the value?

**Review by stefanosu — APPROVED** [review]

File: bloggies_frontend/src/Forms/BlogForm/index.tsx [hunk 2]
```diff
@@ -14,17 +14,21 @@ interface IProp {
  * `BlogForm` renders a form for creating/editting a blog.
  * (IF EDITING: post and closeModal parameters required)
  */
-function BlogForm({addPost, post, closeModal }: IProp) {
+function BlogForm({ addPost, post, closeModal }: IProp) {
   const membershipStatus = useSelector(
     (st: CustomReduxState) => st.user.membership_status
   );
-  const INITIAL_FORM_VALUES = { title: post?.title || "", description: post?.description || "", body: post?.body || "", is_premium: membershipStatus === 'active'}
+  const INITIAL_FORM_VALUES = { title: post?.title || "", description: post?.description || "", body: post?.body || "", is_premium: false };
```
Comment (megmcdevitt) [inline bloggies_frontend/src/Forms/BlogForm/index.tsx]: For editing the blog, can we do a check for post.is_premium so the box will automatically be checked if it's a premium post?

**Review by megmcdevitt — CHANGES_REQUESTED** [review]
Great! Just one little thing to add.

File: bloggies_frontend/src/Forms/BlogForm/index.tsx (hunk 2 above)
Comment (dogpineapple) [reply bloggies_frontend/src/Forms/BlogForm/index.tsx]: done!

**Review by megmcdevitt — APPROVED** [review]

**Pull request merged**